content/rest/_request.textile

```diff
@@ -0,0 +1,21 @@
+h6(#request).
+  default: request
+
+bq(definition).
+  default: request(String method, String path, Object params, Object body, Object headers, callback("ErrorInfo":/rest/types#error-info err, "HttpPaginatedResponse":/rest/types#http-paginated-response results)))
+
+Makes a REST request to a provided path. This is provided as a convenience for developers who wish to use bleeding edge REST API functionality that is either not documented or is not yet included in the public API, without having to handle authentication, paging, fallback hosts, MsgPack and JSON support, etc. themselves.
+
+h4. Parameters
+
+- method := either @get@ or @post@.<br>__Type: String__
```

Comment:
RIght, but the reason for that is that the paginated queries were added originally for the methods that return large collections - principally `GET`. That never applied to `PUT`. Now for this generic mechanism we want to be able to support all relevant methods, and support pagination in the case of `GET` at least. So we do need to know that `PUT`` works, although we don't necessarily need to handle `PUT` responses with pagination.